Create a single file operator-install.yaml bundle to install the operator (#467)
infinispan/infinispan-operator: README.md

  ```
  $ make run-local
  ```

> 4. Install the Infinispan operator on OKD from the bundle file (option).

Reviewer: ```suggestion
4. Optionally install the Infinispan operator on OKD from the bundle file.
```

>   * Create a new project or use existing one:

Reviewer: ```suggestion
  * Create a new project or use an existing one:
```

  ```
  $ oc new-project infinispan-operator
  ```
  * Run this command to install the operator:
  ```

>   $ oc apply -n infinispan-operator -f https://raw.githubusercontent.com/infinispan/infinispan-operator/master/deploy/operator-install.yaml

Reviewer 2: I would use the same format for all the `oc apply` commands: below we're using local file and no project specified

  ```
5. Open a new terminal window and create an Infinispan cluster with two nodes:
```
$ oc apply -f deploy/cr/minimal/cr_minimal.yaml
infinispan.infinispan.org/example-infinispan configured
```

> 6. Watch the pods start until they start running.

Reviewer: ```suggestion
6. Watch the pods until they are running.
```

  ```
  $ make run-local
  ```
4. Optionally install the Infinispan operator on OKD from the bundle file.
  * Create a new project or use an existing one:
  ```
  $ oc new-project infinispan-operator
  ```

>   * If your project name different from the `infinispan-operator` (in this example `my-infinispan-operator`), download operator install bundle to the local store and replace namespace with the following command:

Reviewer 2: ```suggestion
  * If your project name differs from `infinispan-operator` (in this example `my-infinispan-operator`), you need to download the operator install bundle file to the local store and replace the predefined namespace with the following command:
```

Reviewer 2: If you're supposing that the user doesn't checkout the whole project, you probably need to provide info on how to get the `cr_minimal.yaml` below in point 5.

Reviewer 2: maybe @oraNod can provide a better rephrase for the param above.

Author: > If you're supposing that the user doesn't checkout the whole project, you probably need to provide info on how to get the `cr_minimal.yaml` below in point 5.

@rigazilla let's point to the raw.githubusercontent.com whether local or remote creating

Reviewer 2: ok

>   * If your project name differs from `infinispan-operator` (in this example `my-infinispan-operator`), you need to download the operator install bundle file locally and replace the predefined namespace with the following command:

Reviewer: ```suggestion
  * If your project name is not `infinispan-operator`, download the Operator install bundle file and replace the pre-defined namespace. For example, the following command changes the project name to `my-infinispan-operator`:
```

Author: Thanks, @oraNod , fixed

```bash
$ make minikube-start
```
3. Build the operator from the source code and run it locally:
```bash
$ make minikube-run-local
```
4. Optionally install the Infinispan operator on Kubernetes from the bundle file.
  * Create a new namespace or use an existing one:
  ```
  $ kubectl create namespace local-operators
  ```
  * Run this command to install the operator:
  ```

>   $ kubectl apply -n local-operators -f https://raw.githubusercontent.com/infinispan/infinispan-operator/master/deploy/operator-install.yaml

Reviewer 2: see comment above.
Could we use also here `infinispan-operator` as namespace?

  ```
5. Open a new terminal window and create an Infinispan cluster with two nodes:
```bash
$ kubectl apply -f deploy/cr/minimal/cr_minimal.yaml -n local-operators
```

> 6. Watch the pods start until the start running

Reviewer: ```suggestion
6. Watch the pods until they are running
```

```bash
$ make minikube-start
```
3. Build the operator from the source code and run it locally:
```bash
$ make minikube-run-local
```
4. Optionally install the Infinispan operator on Kubernetes from the bundle file.
  * Create a new namespace or use an existing one:
  ```

>   $ kubectl create namespace infinispan-operator

Reviewer 2: see comments for the openshift section
